File: Python/Reverse array.py
def reverse_arr(arr):
    i = 0
    j = len(arr) - 1
    while i < j:
        temp = arr[i]
        arr[i] = arr[j]
        arr[j] = temp
        i += 1
        j -= 1


nums = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

print("Array before reversing: ")
for i in nums:
    print(i)

reverse_arr(nums)


print("Array after reversing: ")
for i in nums:
    print(i)

File: JavaScript/Reverse array.js
const reverseArr = (arr) => {
  let i = 0;
  let j = arr.length - 1;

  while (i < j) {
    let temp = arr[i];
    arr[i] = arr[j];
    arr[j] = temp;
    i++;
    j--;
  }
};

let nums = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];

console.log("Array before reversing: ");
for (let i = 0; i < 10; i++) {
  console.log(nums[i]);
}

reverseArr(nums);

console.log("Array after reversing: ");
for (let i = 0; i < 10; i++) {
  console.log(nums[i]);
}
